frontend/apps/reader/modules/readerrolling.lua

```diff
@@ -801,7 +815,7 @@ end
 
 function ReaderRolling:onGotoXPointer(xp, marker_xp)
     if self.mark_func then
-        -- unschedule previous marker as it's no more accurate
+        -- unschedule previous marker as it's no longer accurate
```

Comment:
If you're touching random comments touch 'em up all the way. :-P
```suggestion
        -- Unschedule previous marker as it's no longer accurate.
```